apps/src/levelbuilder/level-editor/EditChildLevelSettings.jsx

```diff
@@ -0,0 +1,161 @@
+//Create a boilerplate for the EditChildLevelSettings component
+//This component will be used to edit the settings of any child levels
+//The component will take in the following props:
+//Child levels: an array of objects representing the child levels
+import PropTypes from 'prop-types';
+import React, {useState, useCallback} from 'react';
+
+import ImageInput from '@cdo/apps/levelbuilder/ImageInput';
+import CollapsibleSection from '@cdo/apps/templates/CollapsibleSection';
+import {getAuthenticityToken} from '@cdo/apps/util/AuthenticityTokenStore';
+
+import BubbleChoiceDescriptionEditor from './BubbleChoiceDescriptionEditor';
+
+import styles from './edit-child-level-settings.module.scss';
+
+//Create a boilerplate for the EditChildLevelSettings React component
+const EditChildLevelSettings = ({initialChildLevelSettings}) => {
+  const [childLevelSettings, setChildLevelSettings] = useState(
+    initialChildLevelSettings
+  );
+  const [saveSuccess, setSaveSuccess] = useState(false);
+
+  const handleDisplayNameChange = (index, newDisplayName) => {
+    const updatedSettings = childLevelSettings.map((childLevel, i) => {
+      if (i === index) {
+        return {
+          ...childLevel,
+          properties: {
+            ...childLevel.properties,
+            display_name: newDisplayName,
+          },
+        };
+      }
+      return childLevel;
+    });
+    setChildLevelSettings(updatedSettings);
+  };
+
+  const handleDescriptionChange = useCallback(
+    (index, newDescription) => {
+      const updatedSettings = childLevelSettings.map((childLevel, i) => {
+        if (i === index) {
+          return {
+            ...childLevel,
+            properties: {
+              ...childLevel.properties,
+              bubble_choice_description: newDescription,
+            },
+          };
+        }
+        return childLevel;
+      });
+      setChildLevelSettings(updatedSettings);
+    },
+    [childLevelSettings]
+  );
+
+  const handleThumbnailUrlChange = (index, newThumbnailUrl) => {
+    const updatedSettings = childLevelSettings.map((childLevel, i) => {
+      if (i === index) {
+        return {
+          ...childLevel,
+          properties: {
+            ...childLevel.properties,
+            thumbnail_url: newThumbnailUrl,
+          },
+        };
+      }
+      return childLevel;
+    });
+    setChildLevelSettings(updatedSettings);
+  };
+
+  async function handleSave(index) {
+    event.preventDefault();
+    const childLevel = childLevelSettings[index];
+    // Here you would typically make an API call to save the changes for the specific child level
+    console.log(`Saving changes for child level ${index}:`, childLevel);
+    // Example API call (uncomment and replace with actual API endpoint)
+    const url = `/levels/${childLevel.id}/update_bubble_choice_settings`;
```

Comment:
Just a checkin, is this comment stale or does this indicate we're missing an edge case here? Could be fine to merge with this unhandled, but just want to make sure we're not missing something important.